packages/embark-utils/README.md

```diff
@@ -0,0 +1,6 @@
+# `embark-graph`
+
+> Smart contract documentation generator for Embark DApps
```

Comment:
Needs an appropriate description that matches what's in `package.json` description.